vector/src/main/res/values/strings.xml

```diff
@@ -3263,7 +3266,7 @@
     <string name="loading_contact_book">Retrieving your contacts…</string>
     <string name="empty_contact_book">Your contact book is empty</string>
     <string name="contacts_book_title">Contacts book</string>
-    <string name="phone_book_perform_lookup">Search for contacts on Matrix</string>
+    <string name="template_phone_book_perform_lookup">Search for contacts on ${app_name}</string>
```

Comment:
Can you please revert this change? the request #248 has been cancelled for the moment